modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,58 @@
+= Accessing the console
+
+In order to view the {app-security-service} Admin console open the URL in a browser. The URL can be retrieved from the OpenShift console. 
```

Comment:
`The URL can be retrieved from the OpenShift console.` needs to go, suggestions welcome

Reply:
after binding, you can nav from the mobile client (or whatever it's called these days)?

Reply:
I've modified this @finp & switched from using `console` to `dashboard` to be more consistent with the other mobile services.

Reply:
@damienomurchu  not sure if we're very consistent, ping me to discuss